### PR TITLE
Toast

### PR DESCRIPTION
**Pull request by muskan-gupta-0803**
Added toast component and docs

### CODE REVIEW

File: components/Toast/toast.css [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+.toast-right{
+    justify-content: end;
+}
+
+.toast-btn{
+    display: flex;
+    flex-shrink: 6;
+    width: 5rem;
+    min-width: 20x;
```
Comment (sanjay-xdr) [inline components/Toast/toast.css]: what is 20x?

**Review by sanjay-xdr — COMMENTED** [review]
Everthing looks good

File: components/Toast/toast.css (hunk 1 above)
Comment (muskan-gupta-0803) [reply components/Toast/toast.css]: corrected

File: components/Toast/toast.css [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+@import "../Utilities/utilities.css";
+@import "../Button/button.css";
+
+
+.toast{
+    display: flex;
+    align-items: center;
+    justify-content: space-evenly;
+    width:60%;
+    border:1px solid #808080;
```
Comment (pritam-kr) [inline components/Toast/toast.css:11]: use variables for colors

File: components/Toast/toast.css [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+
+}
+
+.toast-pos{
+    display: flex;
+    align-items: flex-end;
+    width:100%;
+    height: fit-content;
+    position: absolute;
+    bottom: 20px;
```
Comment (pritam-kr) [inline components/Toast/toast.css:24]: you can use rem instead PX unit

**Review by pritam-kr — APPROVED** [review]
All looks neat and clean

- your code is formatted 
- used semantic HTML
- commented for code is also good
- I have added some points you can work on these..

otherwise all looks good to merge

File: components/Toast/toast.css (hunk 2 above)
Comment (muskan-gupta-0803) [reply components/Toast/toast.css:11]: sure

File: components/Toast/toast.css (hunk 3 above)
Comment (muskan-gupta-0803) [reply components/Toast/toast.css:24]: sure

**Comment by muskan-gupta-0803** [comment]
> All looks neat and clean
> 
> * your code is formatted
> * used semantic HTML
> * commented for code is also good
> * I have added some points you can work on these..
> 
> otherwise all looks good to merge

Thankyou .
also, thanks for reviewing.

**Pull request merged**